kaptive/2.0.5/Dockerfile

```diff
@@ -0,0 +1,71 @@
+FROM ubuntu:focal as app
+
+ARG KAPTIVE_VER="2.0.5"
+ARG blastVer="2.3.0"
+# so that apt doesn't try to ask for user input
+ARG DEBIAN_FRONTEND=noninteractive
+
+LABEL base.image="ubuntu:focal"
+LABEL dockerfile.version="1"
+LABEL software="Kaptive"
+LABEL software.version="${KAPTIVE_VER}"
+LABEL description="Report information about surface polysaccharide loci for Klebsiella pneumoniae species complex and Acinetobacter baumannii genome assemblies"
+LABEL website="https://github.com/katholt/Kaptive"
+LABEL license="https://github.com/katholt/Kaptive/blob/master/LICENSE"
+LABEL maintainer="Tamas Stirling"
+LABEL maintainer.email="stirling.tamas@gmail.com"
+LABEL maintainer2="Curtis Kapsak"
+LABEL maintainer2.email="kapsakcj@gmail.com"
+
+# install some dependencies via apt; cleanup apt garbage
+RUN apt-get update && apt-get install -y \
+  locales \
+  python3 \
+  python3-pip \
+  python3-setuptools \
+  software-properties-common \
+  wget \
+  --no-install-recommends && \
+  rm -rf /var/lib/apt/lists/* && apt-get autoclean
+
+# set locale
+RUN locale-gen en_US.UTF-8 && update-locale LANG=en_US.UTF-8
+ENV LANG=en_US.UTF-8
```

Comment:
Is this general good practice or something that kaptive needs?

Reply:
I think it's specifc to kaptive python code, not sure if I'd advise doing this for all dockerfiles. I don't know enough of the details about the variable and implications of changing it